src/app/utils/item.js

```diff
@@ -138,8 +134,9 @@ function LibraryItem() {
       suppressed,
       barcode,
       itemSource: mappedItemSource,
-      isRecap: isRecap || isOffsite,
```

Comment:
so we still need `isRecap` and` isOffsite`, but they will be put in different fields?